code/backend/batch/utilities/helpers/embedders/PushEmbedder.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from typing import List
+
+from ..AzureBlobStorageClient import AzureBlobStorageClient
+
+from ..config.EmbeddingConfig import EmbeddingConfig
+from ..config.ConfigHelper import ConfigHelper
+
+from .EmbedderBase import EmbedderBase
+from ..AzureSearchHelper import AzureSearchHelper
+from ..DocumentLoadingHelper import DocumentLoading
+from ..DocumentChunkingHelper import DocumentChunking
+from ...common.SourceDocument import SourceDocument
+
+logger = logging.getLogger(__name__)
+
+
+class PushEmbedder(EmbedderBase):
+    def __init__(self, blob_client: AzureBlobStorageClient):
+        self.blob_client = blob_client
+        config = ConfigHelper.get_active_config_or_default()
+        self.processor_map = {}
```

Comment:
Would we ever expect there to be two processors for the same file extension? Because the code supported that previously but now we're losing it.

We could still have it by having a dict where the key is the extension and the value is a list of processors. Could maybe use `defaultdict`

Reply:
I thought about that and I don't see that use case coming up. What will be the distinction for the same extension to have 2 different processors?

Reply:
I don't see a need for it myself